mobile/src/main.jsx

```diff
@@ -19,17 +19,10 @@ import filesApp from './reducers'
 import MobileAppRoute from './components/MobileAppRoute'
 
 import { loadState, saveState } from './lib/localStorage'
-import { initClient, initBar, isClientRegistered, resetClient, refreshFolder } from './lib/cozy-helper'
-import { revokeClient } from './actions/authorization'
+import { initClient, initBar, isClientRegistered, resetClient, refreshFolder, onError } from './lib/cozy-helper'
 
 const loggerMiddleware = createLogger()
 
-const onError = (store, callback) => () => {
-  console.warn(`Your device is no more connected to your server: ${store.getState().mobile.settings.serverUrl}`)
-  store.dispatch(revokeClient())
-  callback()
```

Comment:
I did a mistake by adding `callback()` in this function. So we should remove it from here and call it in the `react-router` function.